content/utils/ui-utils.js

```
@@ -312,261 +324,120 @@ class UIUtils {
-      Object.assign(highlight.style, {
-        top: `${rect.top + window.scrollY}px`,
-        left: `${rect.left + window.scrollX}px`,
-        width: `${rect.width}px`,
-        height: `${rect.height}px`,
-      });
-
-      document.body.appendChild(highlight);
+  removeTemporaryHighlights() {
+    document
+      .querySelectorAll(".highlight-saver-scroll-target")
+      .forEach((el) => this.unwrapMark(el));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)